### DISCUSSION TITLE
FileSystem, not partions found

### DISCUSSION THREAD

**Discussion by TiM-SyStEm** [CosmosOS/Cosmos#1914] [Q&A]
I decided to add support for the file system with this code:
`
CosmosVFS fs = new CosmosVFS();
VFSManager.RegisterVFS(fs);
'
I run it on a virtual machine and here such a error
![SharedScreenshot](https://user-images.githubusercontent.com/72683799/130979790-4f4da7e7-1dbf-41b3-9d42-0f08d98511d0.jpg)
I used virtual box 6.1

**Comment by Toktatop** [comment]
try use vmware
